spec/lib/vanagon/component/source/git_spec.rb

```diff
@@ -29,6 +31,13 @@
       expect { @klass.new(@url, ref: @ref_tag, workdir: @workdir) }
         .to raise_error(Vanagon::InvalidRepo)
     end
+
+    it "uses the realpath of the workdir if we're in a symlinked dir" do
+      expect(File).to receive(:realpath).and_return("/tmp/bar")
```

Comment:
I know we're doing a bit of dodgy path hardcoding here, but is it possible to write this test without hardcoding assumptions about the availability of "/tmp"? I don't have any quick answers here but I might in the morning?